src/installer/tests/HostActivation.Tests/PortableAppActivation.cs

```diff
@@ -554,6 +554,40 @@ public void AppHost_GUI_FrameworkDependent_MissingRuntimeFramework_ErrorReported
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void AppHost_GUI_RuntimeMissing_PrintArchitectureAndRuntimeVersionOnDialog()
+        {
+            var fixture = sharedTestState.PortableAppFixture_Built
+                .Copy();
+
+            string appExe = fixture.TestProject.AppExe;
+            File.Copy(sharedTestState.BuiltAppHost, appExe, overwrite: true);
+            AppHostExtensions.BindAppHost(appExe);
+            AppHostExtensions.SetWindowsGraphicalUserInterfaceBit(appExe);
+
+            string invalidDotNet = SharedFramework.CalculateUniqueTestDirectory(Path.Combine(TestArtifact.TestArtifactsPath, "guiErrors"));
+            Directory.CreateDirectory(invalidDotNet);
+            File.Copy(sharedTestState.BuiltDotNet.DotnetExecutablePath, Path.Combine(invalidDotNet, "dotnet.exe"), true);
```

Comment:
Why do we need to copy over `dotnet`? Is just creating the directory not enough?

Reply:
No need.